app/policies/application_policy.rb

```diff
@@ -0,0 +1,54 @@
+class ApplicationPolicy
+  attr_reader :user, :record
+
+  def initialize(user, record)
+    @user = user
+    @record = record
+  end
+
+  def index?
+    false
+  end
+
+  def show?
+    scope.where(:id => record.id).exists?
```

Comment:
Not sure what this does - think it is just what gets generate automatically - was going to ask you what it is doing...